File: CPP/petshop.cpp
#include <iostream>
#include <string>

using namespace std;

class petshop //class petshop
{
    private:
        string ID;
        string namaproduk;
        string kategoriproduk;
        int hargaproduk;
    
    public:
        petshop() //constructor
        {
            ID = "";
            namaproduk = "";
            kategoriproduk = "";
            hargaproduk = 0;
        };
        petshop(string id, string nama, string kategori, int harga) //constructor
        {
            ID = id;
            namaproduk = nama;
            kategoriproduk = kategori;
            hargaproduk = harga;
        }

        void setID(string id)
        {
            ID = id;
        }
        void setNamaProduk(string id)
        {
            namaproduk = id;
        }
        void setKategoriProduk(string id)
        {
            kategoriproduk = id;
        }
        void setHargaProduk(int id)
        {
            hargaproduk = id;
        }

        string getID() //
        {
            return ID;
        }
        string getNamaProduk()
        {
            return namaproduk;
        }
        string getKategoriProduk()
        {
            return kategoriproduk;
        }
        int getHargaProduk()
        {
            return hargaproduk;
        }

        ~petshop() //destructor
        {
        }
};

File: PHP/petshop.php
<?php
class petshop
{
    private $ID = '';
    private $namaproduk = '';
    private $kategoriproduk = '';
    private $hargaproduk = '';
    private $fotoproduk = '';

    public function __construct($ID = '', $namaproduk = '', $kategoriproduk = '', $hargaproduk = '', $fotoproduk = '') //parameter
    {
        $this->ID = $ID;
        $this->namaproduk = $namaproduk;
        $this->kategoriproduk = $kategoriproduk;
        $this->hargaproduk = $hargaproduk;
        $this->fotoproduk = $fotoproduk;
    }
    //setter
    public function setID($id)
    {
        $this->ID = $id;
    }
    public function setNamaproduk($nama)
    {
        $this->namaproduk = $nama;
    }
    public function setKategoriproduk($kategori)
    {
        $this->kategoriproduk = $kategori;
    }
    public function setHargaproduk($harga)
    {
        $this->hargaproduk = $harga;
    }
    public function setFotoproduk($foto)
    {
        $this->fotoproduk = $foto;
    }
    //getter
    public function getID()
    {
        return $this->ID;
    }
    public function getNamaproduk()
    {
        return $this->namaproduk;
    }
    public function getKategoriproduk()
    {
        return $this->kategoriproduk;
    }
    public function getHargaproduk()
    {
        return $this->hargaproduk;
    }
    public function getFotoproduk()
    {
        return $this->fotoproduk;
    }
}
?>